### ISSUE TITLE
Doc mode test environment need to be initialized with `surface`

### ISSUE THREAD

**Issue by Flrande** [toeverything/blocksuite#4952]
The current implementation does not include `surface`, which means that some bugs caused by `surface` but not belonging to edgeless are not covered by tests.

For example, #4945 does have [a corresponding test](https://github.com/toeverything/blocksuite/blob/1ce1cfd2f8923bb4730832460653674396bfd146/tests/selection/native.spec.ts#L1553-L1564), but because there is no `surface` in the test environment, it was not detected.

In summary, we need to standardize the initialization state of the test environment. (see init functions in [misc.ts](https://github.com/toeverything/blocksuite/blob/1ce1cfd2f8923bb4730832460653674396bfd146/tests/utils/actions/misc.ts))

**Comment by doodlewind** [comment]
This is now actually considered as a feature, which proves that the doc mode editor should work without `affine:surface` or the edgeless part :)
